engine/lib/friends.php

```diff
@@ -87,6 +87,7 @@ function _elgg_friends_setup_user_hover_menu($hook, $type, $return, $params) {
 			// Always emit both to make it super easy to toggle with ajax
 			$return[] = \ElggMenuItem::factory(array(
 				'name' => 'remove_friend',
+				'data-toggle' => 'add_friend',
```

Comment:
Let's use `data-toggleable-item` or something of that nature. We may want to reserve `data-toggle` for `rel="toggle"` refactor. Or use class.